package.json

```diff
@@ -28,6 +28,7 @@
     "fsevents": "^2.3.3"
   },
   "devDependencies": {
+    "@conciergerie.dev/select-a11y": "^3.6.0",
```

Comment:
This one is move to dependency and use the @datagouv package. It should be remove here